php/libraries/Database.class.inc

```diff
@@ -165,11 +165,20 @@ class Database
 
         }
 
-        $this->_PDO = new PDO(
-            "mysql:host=$host;dbname=$database;charset=UTF8",
-            $username,
-            $password
-        );
+        try {
+            $this->_PDO = new PDO(
+                "mysql:host=$host;dbname=$database;charset=UTF8",
+                $username,
+                $password
+            );
+        } catch (PDOException $e) {
+            error_log(
+                'Invalid credentials supplied in the config file. Could not '
```

Comment:
a PDOException doesn't mean that there were invalid credentials, it just means that it couldn't connect.